frontend/src/components/VerticalWord/CrosswordBoxContainer.tsx

```diff
@@ -67,10 +83,11 @@ const CrosswordBoxContainer = ({ crossword }: CrosswordBoxContainerProps) => {
 
   return (
     <CrosswordContainer>
-      {grid.map( (row, i ) => 
+      {grid.map( (row : any, i : number) => 
         <CrosswordRow key={i}>
-          {row.map( (column, j) =>{ 
+          {row.map( (column : any, j : number) =>{ 

```

Comment:
column here would be a string so we can just use string for now. although if we arent using the variable the in JS common practice is to name it _

Reply:
If if has a meaning, I prefer to name it now rather than later in case we use it